### PR TITLE
update grapevine gateway fees

### PR DESCRIPTION
**Pull request by nickpell**
http://manage.dimagi.com/default.asp?158674

### CODE REVIEW

File: corehq/apps/smsbillables/management/commands/bootstrap_grapevine_gateway_update.py [hunk 1]
```diff
@@ -0,0 +1,67 @@
+from decimal import Decimal
+import logging
+from django.core.management.base import LabelCommand
+from corehq.apps.accounting.models import Currency
+
+from corehq.apps.grapevine.api import GrapevineBackend
+from corehq.apps.sms.models import INCOMING, OUTGOING
+from corehq.apps.smsbillables.models import SmsGatewayFee, SmsGatewayFeeCriteria
+from corehq.apps.smsbillables.utils import get_global_backends_by_class
```
Comment (dimagimon) [inline corehq/apps/smsbillables/management/commands/bootstrap_grapevine_gateway_update.py]: F401 'get_global_backends_by_class' imported but unused

File: corehq/apps/smsbillables/management/commands/bootstrap_grapevine_gateway_update.py [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+    # Outgoing message from arbitrary country
+    SmsGatewayFee.create_new(
+        GrapevineBackend.get_api_id(), OUTGOING, Decimal('0.90'),
+        currency=currency,
+        fee_class=sms_gateway_fee_class,
+        criteria_class=sms_gateway_fee_criteria_class,
+    )
+    logger.info("Updated Global Grapevine gateway fees.")
+
+class Command(LabelCommand):
```
Comment (dimagimon) [inline corehq/apps/smsbillables/management/commands/bootstrap_grapevine_gateway_update.py]: E302 expected 2 blank lines, found 1

**Comment by gcapalbo** [comment]
I think we need to also have entries for Lesotho explicitly otherwise it won't show up in the new pricing table.  Otherwise, looks good.

**Comment by nickpell** [comment]
thanks, updated.

**Comment by gcapalbo** [comment]
ok, ready to merge from my side - did you add the hold off on merge label because we're waiting on the go ahead from the ticket?

**Comment by nickpell** [comment]
yeah

**Comment by nickpell** [comment]
@gcapalbo good to go

**Pull request merged**